src/services/jobsHttpClient.adapter.ts

```diff
@@ -0,0 +1,47 @@
+import { API_URL } from "../constants/constants";
+import { IJob } from "./../domain/Job";
+import httpClientAdapter from "./httpClient.adapter";
+
+export const getJobs = (): Promise<IJob[]> => {
```

Comment:
thanks for adding code, but do you need these logic for this issue?
pls add minimum code for your issues.

issue name and PR content should match in working environment, so let's follow same approach here too!